zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/broker/request/group/BrokerGroupUpdateRequest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.impl.broker.request.group;
+
+import io.camunda.zeebe.broker.client.api.dto.BrokerExecuteCommand;
+import io.camunda.zeebe.protocol.Protocol;
+import io.camunda.zeebe.protocol.impl.record.value.group.GroupRecord;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.GroupIntent;
+import org.agrona.DirectBuffer;
+
+public class BrokerGroupUpdateRequest extends BrokerExecuteCommand<GroupRecord> {
+
+  private final GroupRecord requestDto = new GroupRecord();
+
+  public BrokerGroupUpdateRequest(final long groupKey) {
+    super(ValueType.GROUP, GroupIntent.UPDATE);
+    setPartitionId(Protocol.DEPLOYMENT_PARTITION);
+    request.setKey(groupKey);
```

Comment:
🔧 If I'm not wrong the key should be sets into the requestDto also, can you double check ? I think in the processor we are using the groupKey from the record itself

Reply:
That's a good point. Yes, we get it from the record value. I will set it to both places to be safe and consistent.

Reply:
Another point, we should set the partition after set the key in the request. Otherwise the partition might be override by the `setKey` method